doc/reference.md

```diff
@@ -0,0 +1,23 @@
+# Reference
```

Comment:
Rename file to `useage`.

# Useage

## ACL
The ACL stores permissions and enables developers to grant, revoke, set and check if
user has permissions. Read more about [permissions](http://aurelia-acl.spoonx.org/permissions.html).

Only `isAllowed` is not chainable, this will return a boolean.

### .setPermissions(permissions)  ⇒ `acl`
Set the permissions.

```js
acl.setPermissions({
  messages: ['read', 'write', 'delete']
});
```

### .permit(resource, rules)  ⇒ `acl`
Add additional permissions.

```js
acl.permit(messages, 'edit');
```

### .isAllowed(resource, action) ⇒ `boolean`
Checks if the provided action is allowed.

```js
acl.isAllowed(messages, 'write');
```